lib/trello_wrapper.rb

```diff
@@ -19,6 +19,7 @@
 class TrelloWrapper < TrelloService
 
   attr_accessor :board
+  attr_accessor :client
```

Comment:
I think this line is not needed, Ruby will always call the method as it has the same name and is defined later.